### PR TITLE
Fixed #735.

### LINKED ISSUE #735: webpacked mqtt doesn't contain Store
I create webpack using the following  approach. 
https://github.com/mqttjs/MQTT.js#webpack
It works well.

However, when I call `new mqtt.Store()` then I got the following error.

```
test.html:8 Uncaught TypeError: mqtt.Store is not a constructor
```

It works well on nodejs.

I analyze `mqtt` object using chrome debugger.
It seems that `mqtt` isn't contained `Store` on its top level. It contains `MqttClient`, `connect`, and so on.
If I execute the same program on nodejs, `mqtt` contains `Store`.
I found `Store` in `mqtt`, but it is located on deep level. I attached the file of `mqtt` structure.
mqtt

I try to fix it but I don't know why `Store` is not contained, so far.

### PR DESCRIPTION
**Pull request by redboltz**
Added Store to index.js.

### CODE REVIEW

**Comment by redboltz** [comment]
I analyzed webpack behavior. 
It seems that `index.js`'s exporting code is reflected is webpacked `mqtt`.
So I added `Store` to `index.js`. Then webpacked `mqtt` has `Store` as expected.

**Comment by redboltz** [comment]
`index.js` is referred by webpack because of
https://github.com/mqttjs/MQTT.js/blob/5d4d29513d1b6d5c3dc6e058be4459ec24595b4b/package.json#L60

**Comment by mcollina** [comment]
Can you add a unit test for this?

**Comment by redboltz** [comment]
Could you tell me how do I add and run test for browser?
At travis-ci, I only found node tests.
https://travis-ci.org/mqttjs/MQTT.js/builds/312748264

I found something like tests for browser as follows:
https://github.com/mqttjs/MQTT.js/tree/master/test/browser

I thought that I should add a test `store.js` to the directory, but I don't know how to run it.

**Comment by redboltz** [comment]
I found a way.
The test should check `var mqtt = require('../lib/connect')` has `Store`.

File: test/mqtt_store.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+var mqtt = require('../lib/connect')
+
+describe('store in mqtt', function () {
```
Comment (mcollina) [inline test/mqtt_store.js]: Can you use a more describing name for this? It's not really `mqtt`, it's `connect()` from a Node perspective.

Comment (redboltz) [reply test/mqtt_store.js]: How about this?

```javascript
describe('store in lib/connect/index.js (webpack entry point)', function () {
  it('should create store', function (done) {
    done(null, new mqtt.Store())
  })
})
```

Comment (mcollina) [reply test/mqtt_store.js]: thanks!

Comment (mcollina) [reply test/mqtt_store.js]: That will be perfect, can you push?

**Pull request merged**